CMakeLists.txt

```diff
@@ -87,6 +87,12 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "s390x")
     if(${OQS_DIST_BUILD})
         set(OQS_DIST_S390X_BUILD ON)
     endif()
+elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "riscv")
+    if(BOARD MATCHES "qemu_riscv32")
+        set(ARCH "riscv32")
+    elseif(BOARD MATCHES "qemu_riscv64")
+        set(ARCH "riscv64")
```

Comment:
What in case neither "BOARD" matches? `else` clause worth while adding?

Reply:
added `else` clause